src/calibrate/routes/api.py

```diff
@@ -1,8 +1,8 @@
-base_url = '/api/v1'
+base_url = '/api/v1/calibrate'
 
 route = {
     'root': '/',
-    'calibrate': f'{base_url}/calibrate',
-    'calibrate_tool': f'{base_url}/calibrate_tool',
-    'train_calibrate_tool': f'{base_url}/train_calibrate_tool'
-}
\ No newline at end of file
+    'calibrate': f'{base_url}',
+    'calibrate_tool': f'{base_url}/tool',
+    'train_calibrate_tool': f'{base_url}/train_tool'
```

Comment:
Is it okay to use just `train` instead of `train_tool`? or `/tool/train` to eliminate the underscore

Reply:
> Is it okay to use just `train` instead of `train_tool`? or `/tool/train` to eliminate the underscore

Oh okay, let me do that